### DISCUSSION TITLE
convert RTMDET-ins-tiny to tflite

### DISCUSSION THREAD

**Discussion by shelper** [open-mmlab/mmdetection#11297] [Q&A]
I am trying to convert a finetuned RTMDET-ins-tiny model to tflite for android deployment, and i figured that the model comes with all the logics for NMS and other post processing, and many ops, including all dynamic params, are not supported by TFLITE. 

I wonder if there is a way to remove some layers from the model so it mimics more or less like yolo models that just output all the candidates, so that all the dynamic params can be avoided? if so, how can i remove it after it is trained? or i can just modify the config file to remove those parts, and do a fine tuning and the resulted model will be dynamic params free?

really appreciate if you can provide simple examples, thanks in advance!

**Comment by ramonhollands** [comment]
You can use https://pypi.org/project/onnx-graphsurgeon/ to remove a particular node from the onnx graph
